pkg/controller/persistentvolume/persistentvolume_claim_binder_controller.go

```diff
@@ -193,6 +193,14 @@ func (binder *PersistentVolumeClaimBinder) deleteClaim(obj interface{}) {
 func syncVolume(volumeIndex *persistentVolumeOrderedIndex, binderClient binderClient, volume *api.PersistentVolume) (err error) {
 	glog.V(5).Infof("Synchronizing PersistentVolume[%s], current phase: %s\n", volume.Name, volume.Status.Phase)
 
+	// The PV may have been modified by parallel call to syncVolume, load
+	// the current version.
+	newPv, err := binderClient.GetPersistentVolume(volume.Name)
+	if err != nil {
+		return fmt.Errorf("Cannot reload volume %s: %v", volume.Name, err)
+	}
+	volume = newPv
```

Comment:
Nevermind, I am confusing my controllers. There is no local store use in this one. We use the volumeIndex custom store only.